spec/factories/courses.rb

```diff
@@ -4,5 +4,6 @@
     sequence(:title) { |n| "test course#{n}" }
     sequence(:position) { |n| n }
     identifier_uuid { SecureRandom.uuid }
+    badge_uri { 'badge-foundations.svg' }
```

Comment:
Added to update specs with new field, better alternatives for the dummy file?

Reply:
The only other thing that comes to mind is what we use for the fallback 'icons/odin-icon.svg'.

But it's not a huge deal; foundations should be safe to use.